kubernetes/resource_kubernetes_service_account.go

```diff
@@ -106,12 +107,21 @@ func resourceKubernetesServiceAccountCreate(ctx context.Context, d *schema.Resou
 	if err != nil {
 		return diag.FromErr(err)
 	}
+
 	return resourceKubernetesServiceAccountRead(ctx, d, meta)
 }
 
 func getServiceAccountDefaultSecret(ctx context.Context, name string, config api.ServiceAccount, timeout time.Duration, conn *kubernetes.Clientset) (*api.Secret, error) {
+	b, err := serverVersionGreaterThanOrEqual(conn, "1.24.0")
```

Comment:
Super nitpicky but why did we call this `b` everywhere? I'd have called it `v` or `sv` just cause it's server version

Reply:
Changed to `sv`, `b` was standing for `bool` since `serverVersionGreaterThanOrEqual` returns bool and error, but `sv` makes more sense in this context. Thanks!